tools/test/h5ls/expected/tattr2.ls

```diff
@@ -12,21 +12,30 @@ dset                     Dataset {2/2}
         Type:      8-bit bitfield
     Attribute: bitfield3D {4, 3, 2}
         Type:      8-bit bitfield
+    Attribute: complex {2}
+        Type:      complex number of
+                   IEEE 32-bit little-endian float
+    Attribute: complex2D {3, 2}
+        Type:      complex number of
+                   IEEE 32-bit little-endian float
+    Attribute: complex3D {4, 3, 2}
+        Type:      complex number of
+                   IEEE 32-bit little-endian float
     Attribute: compound {2}
         Type:      struct {
                    "a"                +0    8-bit integer
-                   "b"                +4    IEEE 64-bit little-endian float
-               } 12 bytes
+                   "b"                +8    IEEE 64-bit little-endian float
+               } 16 bytes
```

Comment:
Why did this change?

Reply:
My guess is that the file must have previously been generated on some system where the compiler used different structure padding.